dependency-lint.yml

```diff
@@ -0,0 +1,46 @@
+# array of strings or regular expressions to match against a module name
+# to determine if it is allowed to be unused
+allowUnused:
+  - cucumber
+  - livescript
+
+# array of path patterns to match againt a filename
+# to determine if it is used only for development
+#   (see https://github.com/isaacs/minimatch)
+devFilePatterns:
+  - '{features,spec,test}/**/*'
+  - '**/*_{spec,test}.js'
+
+# array of strings or regular expressions to match against a script name
+# in your `package.json` to determine if it is used only for development
+devScripts:
+  - lint
+  - publish
+  - test
+  - version
+
+# path pattern to match against a filename to determine if it should be parsed.
+# This is the starting point, devFilePatterns and ignoreFilePatterns should be
+# subsets of this pattern
+#   (see https://github.com/isaacs/minimatch)
+filePattern: '**/*.ls'
+
+# array of path patterns to match against a filename
+# to determine if it should be ignored
+#   (see https://github.com/isaacs/minimatch)
+ignoreFilePatterns:
+  - 'node_modules/**/*'
```

Comment:
For what its worth, `.npmignore` by default ignores the `node_modules` folder. It is enough that this is mentioned in the documentation, there is no need to add a comment about this into each `.npmignore` file. 

I think a similar case could be made here. Not linting external dependencies in node_modules is so obviously the right thing to do that we could simply make it a convention here. If somebody isn't sure and adds node_modules here, we could display a warning. 

I simply see no case where node_modules would ever not be here, at least in correctly set up projects.